src/java.desktop/share/classes/javax/swing/plaf/synth/SynthRootPaneUI.java

```diff
@@ -102,6 +108,11 @@ private void updateStyle(JComponent  c) {
                 installKeyboardActions((JRootPane)c);
             }
         }
+
+        if (UIManager.getBoolean("RootPane.altPress")) {
+            KeyboardFocusManager.getCurrentKeyboardFocusManager().
+                    addKeyEventPostProcessor(altProcessor);
+        }
```

Comment:
Can this lead to installing `altProcessor` twice or more?

Reply:
Added flag variable to check of it is installed or not.